docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -209,6 +209,55 @@ def test_my_partitioned_config():
     assert validate_run_config(do_stuff_partitioned, run_config)
 ```
 
+If you want to test that your <PyObject object="PartitionedConfig" /> creates the partitions you expect, you can use the `get_partitions` or `get_partition_keys` or `get_run_config_for_partition_key` functions.
```

Comment:
It would be preferable not to mention `get_partitions`, because we're aiming to remove or at least de-emphasize the `Partition` class.